Plots/Panels/NCL_panel_11.py

```diff
@@ -0,0 +1,172 @@
+"""
+NCL_panel_11.py
+===============
+This script illustrates the following concepts:
+    - Specifying how many and where to draw plots using mosaic_subplot
+
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/panel_11.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/panel_11_1_lg.png and https://www.ncl.ucar.edu/Applications/Images/panel_11_2_lg.png
+"""
+
+###############################################################################
+# Import packages
+
+import numpy as np
+import xarray as xr
+import matplotlib.pyplot as plt
+
+import geocat.datafiles as gdf
+from geocat.viz import util as gvutil
+
+###############################################################################
+# Read in data:
+
+# Open a netCDF data file using xarray default engine
+# and load the data into xarrays
+ds = xr.open_dataset(gdf.get('netcdf_files/chi200_ud_smooth.nc'))
+
+lon = ds.lon
+times = ds.time
+scale = 1000000
+chi = ds.CHI
+chi = chi / scale
+
+###############################################################################
+# Plot:
+
+
+def contour_plot(fig):
+    """Plots a series of contour subplots with position depending on axes in
+    mosaic subplot.
+
+    Parameters
+    ----------
+    fig: :'figure':
+        A figure defined with mosaic_subplot.
+
+    Description
+    -----------
+
+        Takes in a defined figure and plots a contour plot, where each
+        subplot progressively shifts its longitude range east. The function
+        also sets a label beneath the plot, titles and axes labels, and contour
+        labels.
+    """
+
+    # Set the starting longitude for the first subplot
+    l_boundary = 0
+
+    # Create empty list
+    ax_list = []
+
+    for letter in ["A", "B", "C", "D", "E", "F", "G"]:
+        ax_list.append(fig[letter])
+
+    for axis in ax_list:
+        # Set the range of each subplot's longitude
+        h_boundary = l_boundary + 80
+
+        # Draw contour lines at levels [-10, -8, -6, -4, -2, 0, 2, 4, 6, 8, 10]
+        cs = axis.contour(lon,
+                          times,
+                          chi,
+                          levels=np.arange(-10, 12, 2),
+                          colors='black',
+                          linestyles="-",
+                          linewidths=.8)
+
+        # # Label the contour levels -4, 0, and 4
+        cl = axis.clabel(cs, fmt='%d', levels=[-4, 0, 4], fontsize=7)
```

Comment:
```suggestion
        axis.clabel(cs, fmt='%d', levels=[-4, 0, 4], fontsize=7)
```
This doesn't need to be assigned to a variable